config/routes.rb

```diff
@@ -2,7 +2,7 @@
   with_options defaults: { format: :json } do
     root 'pages#index'
 
-    resources :projects, only: [:create, :update, :index, :destroy] do
+    resources :projects, only: [:create, :show, :update, :index, :destroy] do
```

Comment:
I think this now includes everything except `:edit`... time to switch to `resources except: [:edit]`, maybe? Or just remove the whole only/except clause since we're now close enough...